### PR TITLE
Remove bind calls and update packages.

### PR DESCRIPTION
**Pull request by neiloler**

### CODE REVIEW

File: src/multi-selector.js [hunk 1]
```diff
@@ -463,7 +463,7 @@ const MultiSelector = React.createClass({
 						<div
 							style={{ padding: '8px 16px', borderTop: '1px solid rgb(233, 233, 233)' }}
 							className={`cps-bg-gray-3 ${styles['cpr-multi-selector__done']}`}>
-							<a onClick={this.closeDialog.bind(this)} className={`cps-link +primary`}>
+							<a onClick={this.closeDialog} className={`cps-link +primary`}>
```
Comment (internettrans) [inline src/multi-selector.js:466]: Are we getting rid of this just for performance reasons? Or was there a bug related to this?

**Review by internettrans — COMMENTED** [review]
At some point, we should move this component off [its usage of React.createClass](https://github.com/CanopyTax/cpr-multiselect/blob/b1858bbd2db8656357b50403743bc606668bcb11/src/multi-selector.js#L63). The reason is that we can't upgrade to React 16 unless we either migrate off of createClass or start using the create-class npm package instead of calling React.createClass directly.

If you are feeling up to it, that would be a cool change to do in this pr. If not, that's fine we'll get to it at some point

File: src/multi-selector.js (hunk 1 above)
Comment (neiloler) [reply src/multi-selector.js:466]: This is to make happy the warning I was getting: `You are binding a component method to the component. React does this for you automatically in a high-performance way, so you can safely remove this call.`

**Review by internettrans — APPROVED** [review]
🚶 let me know if you need access to publish this

**Pull request merged**